schemas/schema-0.0.0.json

```diff
@@ -0,0 +1,249 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "CMS Code.json Metadata",
+    "description": "A metadata standard for software repositories of CMS",
+    "type": "object",
+    "properties": {
+      "items": {
+          "name": {
+              "type": "string",
+              "description": "Name of the project or software"
+          },
+          "softwareDescription": {
```

Comment:
Flagging this field. I originally named this field as `description` however, this gives me an type error since `description` in the og json schema is defined as a string, not object. 

Not sure how to go about resolving this. For context, we follow what is done on publiccode.yml: https://yml.publiccode.tools/example.html

Reply:
We will go with `softwareDescription` for now!

Reply:
`description` and `longDescription`